### PR TITLE
Wasting treatment 2

### PR DESCRIPTION
**Pull request by Ninicorn**

### CODE REVIEW

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 1]
```diff
@@ -759,3 +589,27 @@ References
...
+
+.. _`Isanaka 2021`: https://gh.bmj.com/content/6/3/e004342
+
+.. [CMAM_Forum_coverage]
+
+  View `CMAM Forum coverage`_ 
+
+    Assessment of Coverage of Community-based Management of Acute Malnutrition
+
+.._`CMAM Forum coverage`: https://www.ennonline.net/assessmentofcmamcoveragev2
```
Comment (NathanielBlairStahn) [inline docs/source/intervention_models/wasting_treatment/index.rst:614]: You're missing a space -- should be ``` .. _`CMAM Forum coverage`: ```

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 2]
```diff
@@ -759,3 +589,27 @@ References
    National Guideline for the Management of Acute
    Malnutrition. Addis Ababa: FMOH.
 
+.. [WHO_2013_SAM_guidelines] 
+  
+  View `WHO 2013 SAM guideline`_
```
Comment (NathanielBlairStahn) [inline docs/source/intervention_models/wasting_treatment/index.rst:594]: Missing an 's' again, should be ``` `WHO 2013 SAM guidelines`_ ```

Comment (Ninicorn) [reply docs/source/intervention_models/wasting_treatment/index.rst:594]: wahh...ok!

**Comment by NathanielBlairStahn** [comment]
This PR replaces https://github.com/ihmeuw/vivarium_research/pull/543, as some of @Ninicorn 's commits didn't make it into that one.

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 3]
```diff
@@ -298,30 +252,61 @@ Congratulate and counsel the mother on appropriate IYCF practices.
   The table below provides example entries for large scale food fortification with iron.
```
Comment (aflaxman) [inline docs/source/intervention_models/wasting_treatment/index.rst:252]: is this line out-of-date?

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 4]
```diff
@@ -298,30 +252,61 @@ Congratulate and counsel the mother on appropriate IYCF practices.
...
+    - Currently we do not have the relative risk of death of treated/untreated. We will capture some of the effect of tx on death by having the simulants recover faster, therefore leave the high risk state faster. However, by assigning utx simulants the mean death rate when they should have a higher death rate, we might overestimate the protective effect of tx on death.
+  * - Proportion recovered within programme treatment length
+    - Increases
+    - No (but we can)
+    - We assume that all treatment covered will recover within the time-to-recovery if they had not died. We can modify the treatment coverage by multiplying it by the 'cure rate' giving us the 'met-need' instead. 1 - 'met-need' will be the 'utx' population. 
+  * - Proportion defaulted during programme
+    - Decreases
+    - We can capture this by using 'met-need'
+    - This will decrease as 'met-need' increases.
+
```
Comment (aflaxman) [inline docs/source/intervention_models/wasting_treatment/index.rst:278]: Maybe Abie and Nicole have diverging understanding of the treatment model and that is making it hard for us to come to consensus on the wasting model.  For discussion!

**Review by aflaxman — APPROVED** [review]
Please try to make smaller commits.  It is hard to review commits this large.

File: docs/source/intervention_models/wasting_treatment/index.rst (hunk 1 above)
Comment (Ninicorn) [reply docs/source/intervention_models/wasting_treatment/index.rst]: thanks!!

File: docs/source/intervention_models/wasting_treatment/index.rst (hunk 4 above)
Comment (Ninicorn) [reply docs/source/intervention_models/wasting_treatment/index.rst:278]: we will get there!! :)

**Review by NathanielBlairStahn — APPROVED** [review]
I haven't thoroughly reviewed everything in this PR, but I'm approving so we can get Nicole's equations merged and use them as a basis for resolving the differences between the treatment models proposed by @Ninicorn and @aflaxman .

**Pull request merged**